src/main/java/io/confluent/connect/jdbc/TimestampIncrementingTableQuerier.java

```diff
@@ -147,18 +149,25 @@ protected void createPreparedStatement(Connection db) throws SQLException {
   @Override
   protected ResultSet executeQuery() throws SQLException {
     if (incrementingColumn != null && timestampColumn != null) {
-      Timestamp ts = new Timestamp(timestampOffset == null ? 0 : timestampOffset);
-      stmt.setTimestamp(1, ts, UTC_CALENDAR);
-      stmt.setLong(2, (incrementingOffset == null ? -1 : incrementingOffset));
-      stmt.setTimestamp(3, ts, UTC_CALENDAR);
-      log.debug("Executing prepared statement with timestamp value = " + timestampOffset + " and incrementing value = " + incrementingOffset);
+      Timestamp startTime = new Timestamp(timestampOffset == null ? 0 : timestampOffset);
+      Timestamp endTime = new Timestamp(new Date().getTime() - timestampDelay);
```

Comment:
Yeah, that will work. It will also mean that if we want to support "ID delay" use-case, it won't be any extra overhead. 
Extra latency is much easier to deal with than time-sync issues, which are a huge time sink (see what I did there?)